src/_ctf-private/ctf-toolbox/CtfToolbox.tsx

```diff
@@ -147,9 +152,20 @@ export const CtfToolbox = () => {
       document.removeEventListener('click', handleClickOutside, true);
       document.removeEventListener('keydown', handleEscape, true);
     };
+  }, [enabled]);
+
+  useEffect(() => {
+    try {
+      if (window.top?.location.href === window.location.href) {
+        // Dont show the settings panel when embedded into an iframe (e.g. live preview)
+        setEnabled(true);
+      }
+    } catch (err) {
+      // window.top.location.href is not accessable for non same origin iframes
```

Comment:
Should you now set enabled to false if you get an error?

Reply:
This is how we're doing it already in the marketing template and it defaults to false, but no harm in making it explicit just in case of any weird edge cases, sure